src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -300,7 +300,7 @@ def _process_resource_change(
 
                 # Trigger a new task for each webhook
                 send_resource_change_webhook.delay(
-                    installation_id=installation.id, event=event, data=data
+                    installation_id=installation.id, event=str(event), data=data
```

Comment:
nit: don't need to cast to str for StrEnum

Reply:
I was getting an error about passing a pickled object without the cast 😭